config/config.exs

```diff
@@ -15,3 +15,17 @@ config :logger, :console,
   metadata: [:request_id]
 
 import_config "#{Mix.env}.exs"
+
+# %% Coherence Configuration %%   Don't remove this line
+config :coherence,
+  user_schema: CoursePlanner.User,
+  repo: CoursePlanner.Repo,
+  module: CoursePlanner,
+  logged_out_url: "/",
+  email_from_name: "Your Name",
+  email_from_email: "yourname@example.com",
+  opts: [:authenticatable, :recoverable, :lockable, :trackable, :unlockable_with_token, :invitable, :registerable]
```

Comment:
@ghatighorias and I were talking about the users today, and they should not be `registerable`, it should be invite only, could you update it to be like that?